distribution/lib/Standard/Database/0.0.0-dev/src/Internal/Postgres/Postgres_Connection.enso

```diff
@@ -263,3 +263,12 @@ type Postgres_Connection
        on the 'subclasses'.
     base_connection : Connection
     base_connection self = self.connection
+
+    ## PRVIATE
+       Returns the collation setting for the current database.
+    is_collation_en_gb_utf8 : Text
```

Comment:
@radeusgd Should this method be added to Connection as well?

Reply:
IMO this method should not be added here at all.

This is a detail of an assumption our _tests_ are making. It's not something that should be part of our library, even as a `PRIVATE` helper, if we can avoid it.

IMO there is no reason to keep this logic inside of the library itself and not just put it in the tests.

Reply:
I've moved it to the test. However, I can only run this check against Postgres -- what's the best way to find out if it's Postges, without modifying the _Connection classes?

Reply:
We were usually checking the prefix which contains the name of the DB:
https://github.com/enso-org/enso/blob/ad7fad42fabb962d200cd3ef1a1c6e66f60a18a2/test/Table_Tests/src/Common_Table_Operations/Filter_Spec.enso#L98-L105